### PR TITLE
[QC-769] Desambiguify DS and Merger names 

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Review by knopers8 — DISMISSED** [review]

File: Utilities/Mergers/src/MergerBuilder.cxx [hunk 1]
```diff
@@ -67,7 +67,7 @@ framework::DataProcessorSpec MergerBuilder::buildSpec()
 {
   framework::DataProcessorSpec merger;
 
-  merger.name = mergerIdString() + "-" + mName + std::to_string(mLayer) + "l-" + std::to_string(mId);
+  merger.name = mergerIdString() + "-" + mConfig.detectorName + "-" + mName + std::to_string(mLayer) + "l-" + std::to_string(mId);
```
Comment (knopers8) [inline Utilities/Mergers/src/MergerBuilder.cxx]: you could consider switching the order to comply with the order in QC

Comment (Barthelemy) [reply Utilities/Mergers/src/MergerBuilder.cxx]: ok

**Comment by Barthelemy** [comment]
@ktf I think that the fullCI error is spurrious. Could you confirm and merge ?

**Comment by ktf** [comment]
Yes, agreed.

@davidrohr @shahor02 since a couple of days I see:

```
## sw/BUILD/O2-full-system-test-latest/log
--
Detected critical problem in logfile digi.log
digi.log-[77046:FDDDigitizer]: [21:34:13][INFO] FDD: Sending ROMode= 3 to GRPUpdater
digi.log-[77050:CPVDigitizer]: [21:34:55][STATE] Exiting FairMQ state machine
digi.log:[77050:CPVDigitizer]: [21:34:55][FATAL] unknown error while setting up workflow in o2-sim-digitizer-workflow.
digi.log-[ERROR] pid 77050 crashed with 1
digi.log-[INFO] QUIT_REQUESTED
killing child 75041
killing child 75044
killing child 75047
killing child 75048
killing child 75052
killing child 75054
killing child 77027
killing child 98226
/sw/slc8_x86-64/O2/8660-slc8_x86-64-local1/share/scripts/jobutils.sh: line 49: 75041 Killed                  eval ${finalcommand} >> ${logfile} 2>&1
Running: TIME="#walltime %e" /sw/slc8_x86-64/O2/8660-slc8_x86-64-local1/share/scripts/monitor-mem.sh /usr/bin/time --output=digi.log_time './digi.log_tmp.sh'
/usr/bin/time --output=digi.log_time ./digi.log_tmp.sh
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
   Missing FileEntry for HepMC3/Data/GenParticleData.h
   requested to autoload type HepMC3::GenParticleData
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
```

was there any relevant change in FDD?

**Pull request merged**